### ISSUE TITLE
Training on camvid dataset

### ISSUE THREAD

**Issue by vietdoan** [e-lab/LinkNet#11]
Hi.
I can't reproduce your result on camvid dataset. What is the learning rate and number of training epoch you used in your training, is your published result on validate or test set?.

**Comment by codeAC29** [comment]
@vietdoan The default learning rate should work and train it for 300 epochs. The published result is on test set.

**Comment by vietdoan** [comment]
Thanks,
Can you specify the mean pixel value of train set you used?

**Comment by songh1024** [comment]
@vietdoan I also can't reproduce the result on camvid dataset.
Could you tell me your experimental results?

**Comment by vietdoan** [comment]
@songh1024  Actually I have reproduced the result of linknet paper, It's my mistake.
